Casks/simplemind.rb

```diff
@@ -0,0 +1,28 @@
+cask "simplemind" do
+  version "1.31.0,b2363"
+  sha256 "caa5cf937137abd060fad08323e9ec753af4a4dcd05b5526e4f918d19be34677"
+
+  url "https://simpleapps.eu/simplemind/SMPOSX1_yQj3MSn64GXSTF36Rtj/SimpleMindMacOS#{version.csv.first.no_dots}#{version.csv.second}.dmg"
+  name "SimpleMind"
+  desc "Is the world leader in cross-platform Mind Mapping tools"
```

Comment:
No marketing talk in the description
```suggestion
  desc "Cross-platform Mind Mapping tool"
```